CONTRIBUTING.md

```diff
@@ -45,34 +45,34 @@ When developing a new component, add it to the `packages/ember-toucan-core` dire
 1. Components are added under `packages/ember-toucan-core/src/components`
 2. When adding a new component, it needs to be added to the template registry so that Glint picks it up. To do that, add a line in `packages/ember-toucan-core/src/template-registry.ts`.
 
-## Running the docs/test-app
-
-Prior to running the `docs-app` or `test-app`, you'll need to install dependencies:
-
-```bash
-cd docs-app
-pnpm i
-pnpm start
-```
-
-```bash
-cd test-app
-pnpm i
-pnpm start
-```
-
 ## Viewing changes in the docs/test-app
 
-[We're working on making this better](https://github.com/CrowdStrike/ember-toucan-core/issues/40), but for the time being you can use `pnpm link` to create a symlink between the addon and the apps. This will allow you to make a change in the addon and immediately see the change in the app rather than having to stop the app, rebuild, resync dependencies, etc.
+For the time being you can use `pnpm link` to create a symlink between the addon and the apps. This will allow you to make a change in the addon and immediately see the change in the app rather than having to stop the app, rebuild, resync dependencies, etc.
 
 ```bash
-# Create a symlink in the addon
+# Create a symlink in the ember-toucan-core addon
 cd packages/ember-toucan-core
+pnpm build
 pnpm link .
 
-# Link the docs-app
+# Create a symlink in the forms addon (required even if you aren't using it)
+cd packages/ember-toucan-form
```

Comment:
Since you've already `cd`d to `packages/ember-toucan-core`, I think this should be: `cd ../ember-toucan-form`